lib/foreman_teamdynamix/version.rb

```diff
@@ -1,3 +1,3 @@
 module ForemanTeamdynamix
-  VERSION = '1.0.0'.freeze
```

Comment:
This should be a major version 2.0.0, since it changes foreman/ruby compatibility and is a breaking change